HeterogeneousCore/SonicTriton/src/TritonData.cc

```diff
@@ -116,14 +117,16 @@ void TritonInputData::toServer(std::shared_ptr<TritonInput<DT>> ptr) {
   //shape must be specified for variable dims or if batch size changes
   data_->SetShape(fullShape_);
 
-  if (byteSize_ != sizeof(DT))
-    throw cms::Exception("TritonDataError") << name_ << " input(): inconsistent byte size " << sizeof(DT)
+  createConverter<DT>();
```

Comment:
to reduce the duplication of `std::any_cast<std::shared_ptr<TritonConverterBase<DT>>>(converter_)`, maybe `createConverter<DT>()` can do this cast and return the `std::shared_ptr<TritonConverterBase<DT>>`? Then this could be:
```cpp
auto converter = createConverter<DT>();
```